bindata/network/ovn-kubernetes/self-hosted/ovnkube-control-plane.yaml

```diff
@@ -135,8 +135,13 @@ spec:
           # will rollout control plane pods as well
           network_segmentation_enabled_flag=
           multi_network_enabled_flag=
-          if [[ "{{.OVN_NETWORK_SEGMENTATION_ENABLE}}" == "true" ]]; then
+          if [[ "{{.OVN_MULTI_NETWORK_ENABLE}}" == "true" ]]; then
             multi_network_enabled_flag="--enable-multi-network"
+          fi
+          if [[ "{{.OVN_NETWORK_SEGMENTATION_ENABLE}}" == "true" ]]; then
```

Comment:
This seems like a logic change. Was there a reason behind it?
@ricky-rav you are good with that right? I vaguely remember you were working on some bug around this.

Reply:
Hmm, not really. Look on configmap part i remove
`{{- if .OVN_MULTI_NETWORK_ENABLE }}
    enable-multi-network=true
{{- end }}`
So in case OVN_MULTI_NETWORK_ENABLE was set control-plane was getting this flag through CM in anycase

Reply:
Thanks for confirming!